fidesctl/src/fidesctl/core/config/api_settings.py

```diff
@@ -15,23 +15,32 @@ class APISettings(FidesSettings):
     """Class used to store values from the 'cli' section of the config."""
 
     # Database
-    test_database_url: str = "postgres:fidesctl@fidesctl-db:5432/fidesctl_test"
-    database_url: str = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
-    sync_database_url: str = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
-    async_database_url: str = "postgres:fidesctl@fidesctl-db:5432/fidesctl"
+    database_user: str = "postgres"
+    database_password: str = "fidesctl"
+    database_host: str = "fidesctl-db"
+    database_port: str = "5432"
+    database_name: str = "fidesctl"
+    test_database_name: str = "fidesctl_test"
+    database_url: Optional[str]
+    sync_database_url: Optional[str]
+    async_database_url: Optional[str]
```

Comment:
Are these left as optional so you can write to a separate db completely in a prod setting?